extensions/mssql/src/objectManagement/objectManagementService.ts

```diff
@@ -432,6 +434,32 @@ export class TestObjectManagementService implements IObjectManagementService {
 		};
 	}
 
+	private getPropertiesView(isNewObject: boolean, name: string): ObjectManagement.PropertiesViewInfo {
+		return isNewObject ? <ObjectManagement.PropertiesViewInfo>{
+			objectInfo: {
+				name: 'Server Properties',
+				language: '',
+				memory: '',
+				operatingSystem: '',
+				platform: '',
+				processors: '',
+				minMemory: '',
+				maxMemory: ''
+			}
+		} : <ObjectManagement.PropertiesViewInfo>{
+			objectInfo: {
+				name: 'Server Properties',
```

Comment:
Are these just temporary dummy values?

Reply:
yup, just for testing the dialogs locally.